docs/sources/dashboards/dashboard-public.md

```diff
@@ -0,0 +1,48 @@
+---
+aliases:
+  - /docs/grafana/latest/dashboards/dashboard-manage/
+  - /docs/grafana/latest/features/dashboard/dashboards/
+title: Public dashboards
+weight: 8
+---
+
+## Public dashboards
+
+> **Note:** This page describes a feature for Grafana 9.1 and is available as an opt-in alpha feature.
```

Comment:
```suggestion
> **Note:** This feature is an opt-in alpha feature.
```
* Docs are versioned, so it is redundant to include the release number.
* "This page" talks about the documentation itself, rather than the feature directly. Remove all references to the documentation layer, which ideally is "invisible" to the reader.